Dockerfile

```diff
@@ -0,0 +1,34 @@
+# image for pg-elixir-tools
+FROM elixir:1.9.4-slim
+
+EXPOSE 4000
+
+## Install the necessary dependencies
+RUN apt-get update -y && \
+    apt-get -y install git make wget && \
+    rm -rf /var/lib/apt/lists/*
+
+## Create directory and isolate it from the rest of the docker for security reaasons
+RUN mkdir /app && \
+    groupadd -r elixir && \
+    useradd -d /app -g elixir elixir && \
+    chown elixir:elixir /app usr/local/lib/
+
+# Build vars
+ARG CODE_VERSION
+
+## Env vars
+WORKDIR /app
+ENV MIX_ENV=prod
+ENV CODE_VERSION=$CODE_VERSION
+
+# Install hex locally
+RUN mix local.hex --force && mix local.rebar --force
+
+# Copy the contents
+COPY . /app/
+
+# Compile
```

Comment:
the content of this file is a bit different from the file in `pg-payments` for example, why that?

Reply:
In this case we don't need the same kind of image as in a regular service. This image is just for developing.

Are you suggesting the `RUN` commands are not necessary here?